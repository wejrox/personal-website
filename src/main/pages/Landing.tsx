import * as React from "react";

class Landing extends React.Component {
    public render() {
        return (<div>Landing</div>);
    }
}

export default Landing;
