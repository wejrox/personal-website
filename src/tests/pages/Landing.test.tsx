import React from "react";
import * as Enzyme from "enzyme";
import Landing from "../../main/pages/Landing";

describe("Landing Page", () => {
    const wrapper = Enzyme.shallow(<Landing/>);

    it("renders without crashing", () => {
        expect(wrapper).toMatchSnapshot();
    });
});
