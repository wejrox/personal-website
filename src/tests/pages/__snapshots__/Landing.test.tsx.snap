// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page renders without crashing 1`] = `ShallowWrapper {}`;
